src/buffer/indexOffset.js

```diff
@@ -0,0 +1,306 @@
+import clone from '../clone';
```

Comment:
can't see anywhere this file is used. should this be included?

Reply:
Good pickup - this was trying the `polygon-offset` lib as a replacement for `jsts`, have removed.